tree/tree/test/TBasket.cxx

```diff
@@ -263,7 +263,6 @@ TEST(TBasket, TestVarLengthArrays)
       saved_t1->GetEntry(idx);
       Int_t expected_elem = idx % 9;
       EXPECT_EQ(expected_elem, saved_elem);
```

Comment:
The compiler is still going to complain that saved_elem is not set (because it does not know about GetEntry).

Reply:
The warning was not about uninitialized use, but rather about the compiler being able to prove an ASSERT could not be triggered, right?

I think that's because, in C++, one can assume the `Int[]` is not-`null`.  However, since we pass the address of `saved_elem` to `SetBranchAddress`, it should back off any uninitialized use error.

At least, it does that for me... I don't recall that showing up in the current Jenkins either?

Reply:
Fair enough, it was indeed about the array.